src/components/VSelect/VSelect.vue

```diff
@@ -138,7 +140,7 @@
         return !this.auto ? items.slice(0, this.lastItem) : items
       },
       isAutocomplete () {
-        return this.autocomplete || this.editable
```

Comment:
Is there a reason this was removed?

Reply:
Yes. It enabled autocomplete when just using editable prop